rust/src/p4.v1.rs

```diff
@@ -739,12 +739,26 @@ pub mod packet_replication_engine_entry {
         CloneSessionEntry(super::CloneSessionEntry),
     }
 }
+/// A backup replica used as a fallback port for a replica.
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct BackupReplica {
+    /// Added in v1.5.0.
```

Comment:
I think we still need to increment the working draft to 1.5.0. @smolkaj Did you see https://github.com/p4lang/p4runtime/pull/507#issuecomment-2432816297? Would you prefer we do a simple PR to bump the version, before accepting this PR?

Reply:
As discussed offline with @smolkaj, Steffen said that he prefers to increment the working draft to 1.5.0